src/plugins/cpptemplate/cpptemplate_delegate.hpp

```diff
@@ -0,0 +1,29 @@
+/**
+ * @file
+ *
+ * @brief Delegate definitions for the `cpptemplate` plugin
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#ifndef ELEKTRA_CPP_TEMPLATE_DELEGATE_HPP
+#define ELEKTRA_CPP_TEMPLATE_DELEGATE_HPP
```

Comment:
Is CPP_TEMPLATE renamed? Otherwise it is dangerous if different files get the same include guard.

Reply:
Yes.